compute/cker/include/cker/Types.h

```diff
@@ -80,17 +80,17 @@ enum class BroadcastableOpCategory : uint8_t
 
 struct PoolParams
 {
-  PaddingValues padding_values;
-  int stride_height;
-  int stride_width;
-  int filter_height;
-  int filter_width;
+  PaddingValues padding_values{0, 0};
+  int stride_height{0};
```

Comment:
I intended
```suggestion
  int stride_height = 0;
```
style for consistency and better readability. But, it is okay!